tutorial-content/content/geographical-unit-data.md

```diff
@@ -1,79 +1,80 @@
 
+When aggregating administrative units, it is important to capture territories with homogeneous features that are relevant to the study. For example, if the weather is relevant for the study, the administrative units should be fairly homogeneous concerning mean temperature and precipitation.
+ 
 ```{seealso}
 [Using Weather Data and Climate Model Output in Economic Analyses of Climate Change](https://academic.oup.com/reep/article/7/2/181/1522753) describes the common pitfalls in translating weather data into geographical unit data.
 ```
+ 
+## Geographic information systems (GIS)
+ 
+Geographic information system (GIS) is a computer system for capturing, storing, analyzing and displaying geographical data, and it has been commonly used for spatial research. GIS refers to the representation of polygons, curves, and points, and their use in data analysis.
 
-## Geographic information systems
-
-Much is made of geographic information system (GIS), but these have
-long since become unnotably common parts of spatial research. GIS
-refers to the representation of polygons, curves, and points, and
-their use in data analysis.
+QGIS is a free and open-source desktop geographic information system application that supports viewing, editing and analysis of geospatial data. ArcGIS is proprietary software for working with maps and geographic data. These systems also allow you to perform many kinds of spatial analysis. Some of the most commonly useful are calculating Zonal Statistics, such as the average of a gridded dataset across each shapefile region, and Merging units, and translating points to polygons with Voronoi Polygons.
 
-Describing shapes requires specialized data formats. The most common
-data format is an ESRI Shapefile, and we will focus on these for this
-tutorial. You may also encounter GDB (common in hydrology), GeoJSON
-(used with web interfaces), KXML (used by Google Earth), and OSM (from
-OpenStreetMap) files. All of these can be converted to Shapefiles,
+Administrative units are often defined and described as shapefiles. A [shapefile](https://www.esri.com/library/whitepapers/pdfs/shapefile.pdf) stores geometry and attribute information for the spatial features in a data set. The geometry for a feature is stored as a shape comprising a vector of coordinates. Area features are represented as closed-loop, double-digitized polygons. The shapes together with data attributes linked to each shape create the representation of geographic data like countries, rivers and lakes.
+ 
+Describing shapes requires specialized data formats. The most common data format is an ESRI Shapefile, and we will focus on these for this tutorial. You may also encounter GDB (common in hydrology), GeoJSON (used with web interfaces), KXML (used by Google Earth), and OSM (from OpenStreetMap) files. All of these can be converted to Shapefiles,
```

Comment:
I think this should go above the preceding paragraph.

Reply:
I saw it as: first we introduce what a shape file is (line 56), and then we talk about more technical things like shape file formats (line 58).

Reply:
Unfortunately, I don't think there's a name for the class of information that is distinct from it's format. "Shapefile" means a file of the ESRI Shapefile format. How about reword the first paragraph to not use the term "shapefile"?

```
The boundaries of administrative units are described in specialized files that stores geometry and attribute information. The geometry for a feature is represented by a vector of coordinates....
```

Reply:
Ah ok, I understand now what you mean - I didn't know that the term shapefile, that now people use so often, started with the format itself 😄 Ok I'll make this change!

Reply:
Also, what should I put as decision questions for this section? I can also leave it empty for now

Reply:
Added these two for the section:

```
- How to incorporate geographical and administrative units in your analysis?
- How to generate weather measures that will correspond to your economic data and geographical regions?
```